packages/payment-processor/src/payment/erc777-stream.ts

```diff
@@ -51,7 +51,7 @@ export async function payErc777StreamRequest(
     flowRate: expectedFlowRate ?? '0',
     receiver: paymentAddress,
     superToken: superToken.address,
-    userData: `0x${paymentReference}`,
+    userData: `0xbeefac${paymentReference}`,
```

Comment:
Can you add a comment or a link to the docs as to why we need this?

Reply:
Yes, I will add a comment with the text I've put in the description of the changes in this pull request.